lib/uploadthing-ui-utils.ts

```
@@ -20,3 +24,20 @@ export function getUploadedAmount(progress: number, fileSize: number) {
+
+// Return the objects value
+export function checkFileObjectKey({
+  str,
+  obj,
+}: {
+  str: FileRouterInputKey | undefined;
+  obj: ExpandedRouteConfig | undefined;
+}) {
+  if (!str || !obj) return null;
+
+  if (obj && typeof obj === "object" && obj.hasOwnProperty(str)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)